packages/apps-config/src/api/spec/index.ts

```diff
@@ -226,7 +226,7 @@ const spec: Record<string, OverrideBundleDefinition> = {
   moonshadow: moonbeam,
   'mybank.network Testnet': mybank,
   neatcoin,
-  nft: unique,
+  unique,
```

Comment:
Would just require a re-order (I think this is where CI is complaining)

Reply:
Yep, fixed